Android-Flutter-Merge.yaml

```
@@ -4,65 +4,68 @@ directories:
 - AC_SOURCE
 steps:
 - componentType: appcircle_activate_ssh_key
-  componentVersion: "1.0.*"
+  componentVersion: 1.0.*
   stepName: Activate SSH Private Key
   enabled: true
   onError: $exit
   webUrl: https://github.com/appcircleio/appcircle-activate-ssh-key-component
   repoUrl: https://github.com/appcircleio/appcircle-activate-ssh-key-component.git
-  commit: 3f323d7
+  commit: 6f58522
   inputs:
   - AC_REPOSITORY_SSH_KEY: $AC_REPOSITORY_SSH_KEY
   outputs:
   - SSH_AUTH_SOCK: SSH_AUTH_SOCK
 - componentType: appcircle_git_clone
-  componentVersion: "1.0.*"
+  componentVersion: 1.0.*
   stepName: Git Clone
   enabled: true
   onError: $exit
   webUrl: https://github.com/appcircleio/appcircle-git-clone-component
   repoUrl: https://github.com/appcircleio/appcircle-git-clone-component.git
-  commit: 6343567
+  commit: aa07fc1
   inputs:
-  - AC_GIT_URL: "$AC_GIT_URL"
-  - AC_GIT_LFS: "false"
-  - AC_GIT_BRANCH: "$AC_GIT_BRANCH"
-  - AC_GIT_COMMIT: "$AC_GIT_COMMIT"
-  - AC_GIT_CACHE_CREDENTIALS: "true"
+  - AC_GIT_URL: $AC_GIT_URL
+  - AC_GIT_COMMIT: $AC_GIT_COMMIT
+  - AC_GIT_BRANCH: $AC_GIT_BRANCH
+  - AC_GIT_TAG: ''
+  - AC_GIT_LFS: 'false'
+  - AC_GIT_SUBMODULE: ''
+  - AC_GIT_CACHE_CREDENTIALS: 'true'
+  - AC_GIT_EXTRA_PARAMS: $AC_GIT_EXTRA_PARAMS
   outputs:
-  - AC_REPOSITORY_DIR: "AC_REPOSITORY_DIR"
+  - AC_REPOSITORY_DIR: AC_REPOSITORY_DIR
 - componentType: appcircle_custom_script
-  componentVersion: "1.0.*"
+  componentVersion: 1.0.*
   stepName: Custom Script
   enabled: true
   onError: $exit
-  webUrl: https://github.com/appcircleio/appcircle-custom-script-component
-  repoUrl: https://github.com/appcircleio/appcircle-custom-script-component.git
-  commit: cbea20f
+  webUrl: https://github.com/appcircleio/appcircle-custom-script
```

You have changed the repo and web urls, but there is no changes the urls. Could you please check again.

```suggestion
 https://github.com/appcircleio/appcircle-custom-script-component
```